espnet/nets/chainer_backend/asr_transformer/e2e.py

```diff
@@ -0,0 +1,328 @@
+# encoding: utf-8
```

Comment:
This function must be at `espnet/espnet/nets/chainer_backend/`.
How about `espnet/espnet/nets/chainer_backend/e2e_asr_transformer.py`?